packages/backend/src/models/dynamodb/helpers.ts

```diff
@@ -17,6 +19,14 @@ export function handleDynamoDBError(error: unknown): never {
       throw new Error('DynamoDB Validation Error: ' + message)
     }
     if (error.code < 5000) {
+      if (message.includes('Throughput exceeds the current capacity')) {
```

Comment:
can i just check how did u tell that the error was 4001 specifically ah cus i couldn't trace the logs or find it online

And super nit: the ref link above should be: https://electrodb.dev/en/reference/errors/#error-classifications cus i couldn't access the link oops

Reply:
they changed the link lol. and the 4001 was a guess